### DISCUSSION TITLE
Adjusting MessageBox alignment

### DISCUSSION THREAD

**Discussion by jdhitsolutions** [tui-cs/Terminal.Gui#2868] [Q&A] [answered]
Can the text alignment be adjusted in the MessageBox? I would love to be able to left justify longer messages.

**Comment by BDisp** [comment]
No, by default the `TextAlignment.Centered` is used. Maybe you can manipulate the `MessageBox` with a desired size and insert spaces at right on the text. If you want multi-line, insert `/n` at end. But I'm not sure if this will work because the empty spaces can be removed. Another option is using the `Dialog` with a custom label where you can define the `TextAlignment.Left`.

**Reply by jdhitsolutions** [reply, marked as answer]
I've never used `Dialog` before. Once I figured out how to show it and get my dimensions set, this will solve my problem. Thanks.
